src/modules/project/utils.ts

```diff
@@ -67,9 +67,10 @@ export function getParcelOrientation(layout: Layout, point: Coordinate, rotation
   return parcels
 }
 
-export async function getImageAsDataUrl(url: string): Promise<string> {
+export async function getImageAsDataUrl(url: string, isTemplate = false): Promise<string> {
   const reader = new FileReader()
-  const res = await fetch(url, { headers: NO_CACHE_HEADERS })
+  const headers = !isTemplate ? NO_CACHE_HEADERS : {}
```

Comment:
We're fetching the template thumbnails from the CDN. 

Fetching from the CDN using the `NO_CACHE_HEADERS` throws a CORS error.